pkgs/applications/graphics/feh/default.nix

```diff
@@ -14,6 +14,8 @@ stdenv.mkDerivation rec {
     hash = "sha256-9NJ6zgQHcFJPmRlqJuCMXcKjLvDPUG+QvKGTJlWvWK4=";
   };
 
+  patches = [ ./0001-Right-click-crash-fix.patch ];
```

Comment:
```suggestion
  patches = [
    # upstream PR: https://github.com/derf/feh/pull/723
    ./0001-Right-click-crash-fix.patch
  ];
```

Reply:
Thank you! I have added link for the PR, and changed the commit message.